src/scales/scale.time.js

```diff
@@ -71,7 +71,7 @@ function arrayUnique(items) {
 
 	for (i = 0, ilen = items.length; i < ilen; ++i) {
 		item = items[i];
-		if (!hash[item]) {
+		if (item !== null && !hash[item]) {
```

Comment:
Ah, it doesn't work because a timestamp can be a negative value. We need to iterate `labels` to filter null.